.docker/entrypoint.sh

```diff
@@ -1,102 +0,0 @@
-#!/bin/bash -x
-
-KIMAI=$(cat /opt/kimai/version.txt)
-echo $KIMAI
-
-function waitForDB() {
-  # Parse sql connection data
-  DATABASE_USER=$(awk -F '[/:@]' '{print $4}' <<< "$DATABASE_URL")
-  DATABASE_PASS=$(awk -F '[/:@]' '{print $5}' <<< "$DATABASE_URL")
-  DATABASE_HOST=$(awk -F '[/:@]' '{print $6}' <<< "$DATABASE_URL")
-  DATABASE_PORT=$(awk -F '[/:@]' '{print $7}' <<< "$DATABASE_URL")
-  DATABASE_BASE=$(awk -F '[/?]' '{print $4}' <<< "$DATABASE_URL")
-
-  re='^[0-9]+$'
-  if ! [[ $DATABASE_PORT =~ $re ]] ; then
-     DATABASE_PORT=3306
-  fi
-
-  echo "Wait for database connection ..."
-  until php /dbtest.php "$DATABASE_HOST" "$DATABASE_BASE" "$DATABASE_PORT" "$DATABASE_USER" "$DATABASE_PASS"; do
-    echo Checking DB: $?
-    sleep 3
-  done
-  echo "Connection established"
-}
-
-function handleStartup() {
-  # set mem limits and copy in custom logger config
-  if [ -z "$memory_limit" ]; then
-    memory_limit=512M
-  fi
-  sed -i "s/memory_limit.*/memory_limit=$memory_limit/g" /usr/local/etc/php/php.ini
-  cp /assets/monolog.yaml /opt/kimai/config/packages/monolog.yaml
```

Comment:
this is required in prod: it makes Kimai (Monolog) log to stdout, so we can a) read everything in docker logs and b) do not write to the container.